zipline/data/_equities.pyx

```diff
@@ -99,13 +99,18 @@ cpdef _compute_row_slices(dict asset_starts_absolute,
         intp_t asset_start_calendar
         intp_t asset_end_calendar
 
+        # Flag to check whether we should raise an error because we don't know
+        # about any of the requested sids.
+        uint8_t any_hits = 0
+
     for i, asset in enumerate(requested_assets):
         if asset not in asset_starts_absolute:
             # This is an unknown asset, leave its slot empty.
             continue
-        else:
```

Comment:
This is just a stylistic change. I found this a bit easier to read when all three lookups were grouped together.